iris-mpc-gpu/src/server/actor.rs

```diff
@@ -3032,7 +3032,7 @@ pub fn generate_luc_records(
 
     // If there are no OR rules, return only the lookback records
     if or_rule_indices.is_empty() {
-        return lookback_records;
+        or_rule_indices = vec![vec![]; batch_size];
```

Comment:
why this change? I guess the final result is the same but it does more work now?

Reply:
I saw your explanation for the third case above, nvm.